### ISSUE TITLE
Release 1.3.0 (or the beta) gem

### ISSUE THREAD

**Issue by tpitale** [datamapper/dm-active_model#3]
@dkubb @solnic @dbussink Would one of you mind releasing the 1.3.0 beta gem? I've been running it from git master, but the dm-rails gem won't install

**Comment by tillsc** [comment]
I'm using it without any problems with rails 4.2 and dm-* from their :master branches. What was the problem?

**Comment by tpitale** [comment]
Right, you have to start installing many things from master, or a specific revision. That's why I'm asking for gem releases.
